.editorconfig

```diff
@@ -8,10 +8,6 @@ end_of_line = lf
 insert_final_newline = true
 charset = utf-8
 
-[*.lua]
```

Comment:
why remove this from editorconfig.  this is useful for who use neovim builtin editorconfig.

Reply:
I think there are no different configs for Lua, so I removed its section. Does the settings in [*] section not work for neovim builtin editorconfig?

Reply:
Because it's covered already in the `[*]` block

Reply:
right . it already in `*` block.